internal/aws/containerinsight/utils.go

```diff
@@ -107,6 +108,7 @@ func getPrefixByMetricType(mType string) string {
 	instanceNetPrefix := "instance_interface_"
 	nodeNetPrefix := "node_interface_"
 	nodeEfaPrefix := "node_efa_"
+	hyperPodNodeHealthStatus := "hyper_pod_node_health_status_"
```

Comment:
Going by the pattern of all our other metrics, they tend to follow the pattern pod_, node_, cluster_ etc. And we also have node_status_* metrics for health metrics of a node.

A better name for this metric might be `node_status_hyper_pod_health`.

Reply:
As discussed offline we are aligned with the current naming